### ISSUE TITLE
Pullers cannot be moved with JAWS

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#444]
While testing #443, I tried to move the pullers with JAWS but I could not.

Unfortunately, I suspect that the KeyboardListener is not working because the `tagName` on the pullers is `button`. JAWS only responds to `click` events on the button, and blocks all other keyboard events from reaching the browser. I confirmed this by seeing that the KeyboardListener `fire` callback is never called when JAWS is active.

I confirmed that this patch will fix the problem. By making the group an "application", it tells JAWS to respond to other key presses. Will this change work?

```ts
Subject: [PATCH] Use AccessibleDraggableOptions to make sure that focusable elements have a role, see  https://github.com/phetsims/scenery/issues/1738
---
Index: js/netforce/view/PullerGroupNode.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/netforce/view/PullerGroupNode.ts b/js/netforce/view/PullerGroupNode.ts
--- a/js/netforce/view/PullerGroupNode.ts	(revision 28e9a1d358811ee368a44809944ca8f26fb5ce34)
+++ b/js/netforce/view/PullerGroupNode.ts	(date 1757979180638)
@@ -29,7 +29,8 @@
       tagName: 'div',
 
       // ARIA attributes for the group
-      ariaRole: 'group'
+      ariaRole: 'application',
+      accessibleRoleDescription: 'group'
       // accessibleName will be passed through providedOptions
     }, providedOptions );

```

Another fix would be to leave the markup the same , but move the grab/release out of the KeyboardListener and into a `click` listener on the puller node "button".

**Comment by samreid** [comment]
The proposal looks good, I applied it to all 3 applicable cases. I do not have JAWS, @jessegreenberg can you please confirm that the items on all 4 screens can now be moved with the keyboard?
